ivi-input-modules/ivi-input-controller/src/ivi-input-controller.c

```diff
@@ -539,13 +539,16 @@ input_ctrl_ptr_set_west_focus(struct seat_ctx *ctx_seat,
     struct input_context *ctx = ctx_seat->input_ctx;
     struct seat_focus *st_focus;
     wl_fixed_t sx, sy;
+    struct weston_coord_surface tmp_s;
 
     if (NULL == view) {
         view = weston_compositor_pick_view(pointer->seat->compositor,
                 pointer->pos);
     } else {
-        weston_view_from_global_fixed(view, pointer->x,
-                        pointer->y, &sx, &sy);
+        /*@discussion about the warning*/
```

Comment:
the complete else blocked can be removed the **sx** and **sy** are not used any longer

Reply:
> the complete else blocked can be removed the **sx** and **sy** are not used any longer

Thank @efriedrich. I updated.